test-cases/gemini/gemini-basic-3h.yaml

```diff
@@ -1,20 +1,20 @@
 test_duration: 500
 n_db_nodes: 1
 n_loaders: 1
-n_monitor_nodes: 1
+n_monitor_nodes: 0
 instance_type_db: 'i3.large'
 
 user_prefix: 'gemini-basic-3h'
-failure_post_behavior: keep
+failure_post_behavior: destroy
```

Comment:
At least for the first runs we should run it with keep.
Then, we need to add a functionality that only in case of failure we keep.
@bentsi I think originally this was the idea behind this param, but somehow it became the general behaviour.

Reply:
agree, added a task in Trello

Reply:
return keep